cli/src/main/java/com/walmartlabs/concord/cli/Run.java

```diff
@@ -100,6 +100,9 @@
     @Option(names = {"-v", "--verbose"}, description = "verbose output")
     boolean verbose = false;
 
+    @Option(names = {"--default-import-version"}, description = "default import version or repo branch")
+    String defaultVersion = "master";
```

Comment:
If we change the server.conf default value to `main`, we should probably change it here too.
I don't think it's going to cause any real issues for the current users - there aren't that many CLI uses at the moment :-)

Reply:
changed to main